core/io/resource.h

```diff
@@ -37,6 +37,7 @@
 #include "core/object/ref_counted.h"
 #include "core/templates/safe_refcount.h"
 #include "core/templates/self_list.h"
+#include "scene/scene_string_names.h"
```

Comment:
`core` headers are not allowed to include stuff from `scene`: #53295

Reply:
Ah I thought it's in scene.
I'll move it to some other file maybe.